commercial/app/controllers/commercial/ContentApiOffersController.scala

```diff
@@ -123,17 +123,22 @@ class ContentApiOffersController(contentApiClient: ContentApiClient, capiAgent:
 
   private def renderNative(format: Format, isMulti: Boolean) = Action.async { implicit request =>
 
-    retrieveContent().map(_.toList.headOption).map(
-      {
-        case None => Cached(componentNilMaxAge){ jsonFormat.nilResult }
-        case Some(content) => {
-          val capiSingle = CapiSingle.fromContent(content)
-          Cached(60.seconds) {
-            JsonComponent(capiSingle)
-          }
+    retrieveContent().map {content =>
```

Comment:
I was thinking more
```scala
retrieveContent().map {
  case Nil                => Cached(componentNilMaxAge){ jsonFormat.nilResult }
  case content if isMulti => Cached(60.seconds) {JsonComponent(CapiMultiple.fromContent(content))}
  case first :: _         => Cached(60.seconds) {JsonComponent(CapiSingle.fromContent(head))}   
}
```

`isMulti` is weird, where has it come from? You know if it is single or multi from the list...